e2e/libs/workload/workload.py

```diff
@@ -31,7 +31,7 @@ def create_storageclass(name):
         try:
             api.create_storage_class(body=manifest_dict)
         except Exception as e:
-            print(f"Exception when create storageclass: {e}")
+            logging(f"Creating storageclass error: {e}")
```

Comment:
Right now you just hard-coded use info log, but for error, you want use warn if it's tolerated.

If it's a real error, just throw out to let the caller to deal with and the call will log with error level then handle it.

Reply:
Removed the try/except blocks.